### ISSUE TITLE
Support all services from Registry automatically

### ISSUE THREAD

**Issue by ruudk** [florianv/symfony-swap#9]
Would be nice if you could select all services that are in the Registry.

**Comment by florianv** [comment]
Yes! This could be done with a factory using Swap's new `Builder` class. http://symfony.com/doc/current/service_container/factories.html

Unfortunately, right now this Bundle only supports Swap 2.x and I'm too busy these days to rewrite it for Symfony 3 and Swap 3 support
